en/snap/platform/overview.md

```diff
@@ -25,25 +25,27 @@ Step by step guide to integrate Snap to your platform of choice, will be explain
 
 ## Shopify
 
-Please complete the following steps:
+Please complete the steps given below:
 
 1. Create an online store with [Shopify](https://shopify.com).
-2. Register to Midtrans account [here](https://account.midtrans.com/register).
-3. Complete the account registration form, or get help by contacting [Midtrans activation team](mailto:activation@midtrans.com) with __SHOPIFY - URL Name__ as a subject header and mention your registered _Midtrans Merchant ID_.
+2. Register to [Midtrans account](https://account.midtrans.com/register).
+3. Complete the account registration form. Contact [Midtrans activation team](mailto:activation@midtrans.com) for help. Use **SHOPIFY - URL Name** as a subject header and mention your registered _Midtrans Merchant ID_.
```

Comment:
> Contact [Midtrans activation team](mailto:activation@midtrans.com) for help

This is like directly instructing to contact at any conditions, which suggest contacting is required. Can you emphasise that if help is needed, then contact, which means contacting is optional.